src/styles/blocks/card.scss

```diff
@@ -0,0 +1,21 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  font-family: Roboto, 'Helvetica Neue', Helvetica, sans-serif;
```

Comment:
Consider using a variable for the font family to ensure consistency across different blocks and to make it easier to change the font in the future if needed. You can define this in your 'variables.scss' file.